test/DvsaDocument/Service/Document/DocumentServiceTest.php

```diff
@@ -155,6 +176,11 @@ public function testCreateSnapshotWithInvalidTemplateThrowsExpectedException()
         $this->mockEntityServiceWithQueryBuilder($qb);
 
         try {
+            // this is intentional
+            /**
+             * @psalm-suppress InvalidArgument
+             * @phpstan-ignore-next-line
+             */
```

Comment:
Is this using an undocumented feature or something here? Just trying to understand what's going on.

Reply:
This test intentionally passes an argument of the wrong type to the function and checks that it throws an exception - both static analysis tools catch it as an error even though it's intentional